tasks/patternpack.js

```diff
@@ -26,8 +26,11 @@ module.exports = function (grunt) {
     // TODO: consider using a flag for the "MODE" of operation (dev|build|release)
     // task: "build",
 
-    // Configures the css preprocessor (sass|less)
-    cssPreprocessor: "sass",
+    // Configure our CSS
+    css: {
+      preProcessor: "sass",     // which preprocessor we should use (sass|less|none)
```

Comment:
We need to change the object that is being validated below.
```
ensureOptions(options, "cssPreprocessor", cssPreprocessorValues);
```
since we are now placing this in the `options.css` object we need to pass that object to the `ensureOptions()` function.
```
ensureOptions(options.css, "cssPreprocessor", cssPreprocessorValues);
```